public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,40 @@
+# list
+
+List file structure of a repo including <abbr>data artifacts</abbr>
```

Comment:
List Git repository contents including files and directories under DVC control (data artifacts)

Reply:
`dvc list` can list simple folder (without `.git`), shouldn't it be pointed

Reply:
I would mention it in the description - that's what it is for - to elaborate and expand